src/main/java/pedido/ItemPedido.java

```diff
@@ -10,7 +10,19 @@ public ItemPedido(Shake shake, int quantidade) {
         this.shake = shake;
         this.quantidade = quantidade;
     }
+public enum TipoTamanho {
+    P,M,G;
+    public final double multiplicador(TipoTamanho tamanho) {
+        if(tamanho == TipoTamanho.P) {
+            return 1.0;
+        } else if (tamanho == TipoTamanho.M) {
+            return 1.30;
+        } else {
+            return 1.50;
+        }
+    }
```

Comment:
na classe do tipo enum, ao criar uma variável multiplicador e seu construtor, é possível adicionar o mesmo em cada um dos objetos conforme abaixo:

public enum TipoTamanho {
    P (1.0), M(1.3), G(1.5);

    public final double multiplicador;

    TipoTamanho(final double multiplicador) {
        this.multiplicador = multiplicador;
    }
}